src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -114,19 +116,50 @@ private IEnumerable<LibraryExport> ExportLibraries(Func<LibraryDescription, bool
                     analyzerReferences.AddRange(libraryExport.AnalyzerReferences);
                 }
 
-                yield return LibraryExportBuilder.Create(library)
+                var builder = LibraryExportBuilder.Create(library);
+                if (_runtime != null && _runtimeFallbacks != null)
+                {
+                    builder.WithRuntimeAssemblyGroups(TrimAssetGroups(libraryExport.RuntimeAssemblyGroups, _runtimeFallbacks));
```

Comment:
Yep. The primary effect here is that Desktop apps are always published "low-fat" because: a) They can never be standalone, because .NET Framework can't be deployed standalone and b) They don't support the full-portable model because we don't have anywhere to hook into the native library loader.